### DISCUSSION TITLE
Manually/locally add entries (title + URL) to an existing feed?

### DISCUSSION THREAD

**Discussion by ekr1** [FreshRSS/FreshRSS#7142] [Q&A] [answered]
I have some feeds where the real feed URL only has the newest N entries, and the web site has no options to query for older ones.

It is relatively simple for me to extract the older entries manually - I have a bunch of URLs and their respective titles (I do not care about further content like a text body; I am also happy to not have a timestamp, or fake one). I now wish to add these entries to the exsting feed in FreshRSS.

Is there a convenient or go-to way to do that? I am aware of the Import/Export functionality, but that seems to work on the feeds themselves (i.e. the URLs that then contain the RSS entries), not the entries. At least I had no luck fabricating an OPML file with my feed entries and importing those - I did not receive an error message, basically nothing happened.

I _guess_ I can also simply temporarily shutdown FreshRSS to avoid conflicts, and write directly into the ENTRY table of the sqlite file, but would prefer not to create hard to trace data issues down the line. ;)

Is there an existing way to do what I wish? I looked and did not find an extension for that - maybe one escaped me?

Thanks in advance!

**Comment by math-GH** [comment, marked as answer]
You could use the web scrapper first. After the initial scrapping you can change the feed into a normal RSS feed.

This idea is from @Frenzie (somewhere you mentioned this idea. Cannot find it anymore)

**Reply by Frenzie** [reply]
I suspect you're referring to https://github.com/FreshRSS/FreshRSS/issues/7079#issuecomment-2528287763

Which comes down to the same thing.

**Reply by ekr1** [reply]
> I suspect you're referring to [#7079 (comment)](https://github.com/FreshRSS/FreshRSS/issues/7079#issuecomment-2528287763)

That solution works very well. Exporting the feed, giving a good hard stare at the JSON generated, and fabricating new entries. Easy peasy, not sure how I didn't see that possibility to export actual articles in the Import/Export page. Not enough coffee!
